packages/block-editor/src/text-input/index.js

```diff
@@ -9,6 +9,7 @@ import { __ } from '@wordpress/i18n';
 import { TextInputIcon } from '@crowdsignal/icons';
 import attributes from './attributes';
 import Edit from './edit';
+import Variations from './variations';
```

Comment:
Capitalized variables are usually reserved for and suggest components and/or classes but we're just dealing with a regular object here. I'd suggest we just rename it to `variations`.

Reply:
calling it "variations" was messing with it loading the attribute properly.  I went ahead to change the import to variation to keep in line with the conventions